Source/Core/Core/HLE/HLE_OS.cpp

```diff
@@ -43,7 +43,7 @@ void HLE_GeneralDebugPrint()
 
   NPC = LR;
 
-  NOTICE_LOG(OSREPORT, "%08x->%08x| %s", LR, PC, report_message.c_str());
+  NOTICE_LOG(OSREPORT, "%08x->%08x| %s", LR, PC, SHIFTJISToUTF8(report_message).c_str());
```

Comment:
What is the rule here? is the std::string temporary returned by SHIFTJISToUTF8 required to stick around until NOTICE_LOG is called? if not, just break it out into explicit var and c_str() that.